docs/adr/adr-006-fraud-service.md

```diff
@@ -94,40 +94,44 @@ message BadEnconding {
 }
 ```
 
-`das.Daser` imports a data structure that implements `proof.FraudNotifier` interface that uses libp2p.pubsub under the hood:
+`das.Daser` imports a data structure that implements `proof.Broadcaster` interface that uses libp2p.pubsub under the hood:
 
 ```go
-// FraudNotifier is a generic interface that sends a `Proof` to all nodes subscribed on the FraudNotifier's topic.
-type FraudNotifier interface {
-   // Notify takes a fraud `Proof` data structure that implements standard BinaryMarshal interface and broadcasts it to all subscribed peers.
-   Notify(ctx context.Context, p Proof)  
+// Broadcaster is a generic interface that sends a `Proof` to all nodes subscribed on the Broadcaster's topic.
+type Broadcaster interface {
+   // Broadcast takes a fraud `Proof` data structure that implements standard BinaryMarshal interface and broadcasts it to all subscribed peers.
+   Broadcast(ctx context.Context, p Proof)  
 }
 ```
 
 ```go
 // FraudProofType is a enum type that represents a particular type of fraud proof.
-type FraudProofType string
+type ProofType string
```

Comment:
`Change the docs for this from `FraudProofType` --> `ProofType`

Reply:
done